### PR TITLE
docker: add docker images example

### PR DESCRIPTION
**Pull request by Raizo62**
Hi

In pages.fr/common/docker.md,  add how to display the images already downloaded : 
`docker images`

### CODE REVIEW

**Comment by Raizo62** [comment]
I believe that this command is usefull.
I must remove a other command ? Which ? the first (`docker ps`) because it is include in the second (`docker ps -a`)

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @Raizo62, we have 2 issues here:
1. For us, the translation pages are just translation from the main English pages. Because it's harder to maintain, and keep in sync if every language adds editions. So, you should send your edit in the main page first.
2. We have an strict rule that all pages can have up to 8 examples.

I think `docker images` is a basic command, and for this page we can merge `ps` and `ps -a` examples to add the `images` example.

**Comment by sbrl** [comment]
We already have a more detailed page for `docker images` here: <https://github.com/tldr-pages/tldr/blob/main/pages/common/docker-images.md>

The page for the top-level `docker` commands is one of those pages that has *far* too much functionality to completely represent in a single page (similar to `git` for example), so we have sub-pages for each subcommand thereof.

**Comment by Raizo62** [comment]
> We already have a more detailed page for docker images here: https://github.com/tldr-pages/tldr/blob/main/pages/common/docker-images.md

Yes, but we look the page "docker-images.md" only if we already know that we must use the parameter "images" to display the list of downloaded images. This is not obvious.

**Review by blueskyson — APPROVED** [review]
Look good to me.

File: pages/common/docker.md [hunk 1]
```diff
@@ -4,10 +4,6 @@
 > Some subcommands such as `docker run` have their own usage documentation.
 > More information: <https://docs.docker.com/engine/reference/commandline/cli/>.
 
-- List currently running docker containers:
-
-`docker ps`
-
 - List all docker containers (running and stopped):
 
 `docker ps -a`
```
Comment (navarroaxel) [inline pages/common/docker.md:9]:
```suggestion
`docker ps --all`
```

File: pages.fr/common/docker.md [hunk 2]
```diff
@@ -4,10 +4,6 @@
 > Certaines commandes comme `docker run` ont leur propre documentation.
 > Plus d'informations : <https://docs.docker.com/engine/reference/commandline/cli/>.
 
-- Liste les conteneurs Docker en cours d'exécution :
-
-`docker ps`
-
 - Liste tous les conteneurs Docker (en cours d'exécution ou arrêtés) :
 
 `docker ps -a`
```
Comment (navarroaxel) [inline pages.fr/common/docker.md:9]:
```suggestion
`docker ps --all`
```

**Review by navarroaxel — APPROVED** [review]

**Review by einverne — APPROVED** [review]
Looks good, thanks.

**Review by CleanMachine1 — APPROVED** [review]

**Pull request merged**